pkgs/development/python-modules/atpublic/default.nix

```diff
@@ -0,0 +1,29 @@
+{ stdenv, fetchPypi, buildPythonPackage }:
+
+# python3 only
+buildPythonPackage rec {
+  pname = "atpublic";
+  version = "1.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0i3sbxkdlbb4560rrlmwwd5y4ps7k73lp4d8wnmd7ag9k426gjkx";
+  };
+
+  # TypeError: calling <function public at 0x7ffff26dbb70> returned None, not a test
+  doCheck = false;
+
+  meta = with stdenv.lib; {
+    homepage = "https://public.readthedocs.io/en/latest/";
+    description = "@public for populating __all__";
```

Comment:
missing

Reply:
Missing?

Reply:
Sorry. I though that was some kind of placeholder. It is obviously not but needs to be changed to a real description.

Reply:
Ah okay, I just copied the description from `setup.py` which was `public -- @public for populating __all__`.